config/v1/types_console.go

```diff
@@ -26,13 +26,20 @@ type Console struct {
 type ConsoleSpec struct {
 	// +optional
 	Authentication ConsoleAuthentication `json:"authentication"`
+
+	// +optional
+	Helm HelmConfig `json:"helm"`
 }
 
 // ConsoleStatus defines the observed status of the Console.
 type ConsoleStatus struct {
 	// The URL for the console. This will be derived from the host for the route that
 	// is created for the console.
 	ConsoleURL string `json:"consoleURL"`
+
+	// Status of Helm endpoints
+	// +optional
+	Helm string `json:"helm"`
```

Comment:
What do you intend to provide in the `string` here?

Reply:
I have removed the field, after discussing it with you. The idea was to have a place where we can report Helm endpoint operation status, but learned from you that we have such possibility the resource managed by `console-operator`.